From: Barry Song <v-songbaohua@oppo.com>
Subject: mm: fix the missing doc for flags of folio_add_new_anon_rmap()
Date: Thu, 20 Jun 2024 08:57:55 +1200

Fix mm/rmap.c:1413: warning: Function parameter or struct member
'flags' not described in 'folio_add_new_anon_rmap'.

Link: https://lkml.kernel.org/r/20240619210641.62542-1-21cnbao@gmail.com
Reported-by: kernel test robot <lkp@intel.com>
Closes: https://lore.kernel.org/oe-kbuild-all/202406200059.tVU9semU-lkp@intel.com/
Signed-off-by: Barry Song <v-songbaohua@oppo.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/rmap.c |    1 +
 1 file changed, 1 insertion(+)

--- a/mm/rmap.c~mm-extend-rmap-flags-arguments-for-folio_add_new_anon_rmap-fix-2
+++ a/mm/rmap.c
@@ -1401,6 +1401,7 @@ void folio_add_anon_rmap_pmd(struct foli
  * @folio:	The folio to add the mapping to.
  * @vma:	the vm area in which the mapping is added
  * @address:	the user virtual address mapped
+ * @flags:	The rmap flags
  *
  * Like folio_add_anon_rmap_*() but must only be called on *new* folios.
  * This means the inc-and-test can be bypassed.
_
